config.go

```diff
@@ -1063,6 +1066,24 @@ func loadConfig() (*config, []string, error) {
 		}
 	}
 
+	// peer idle time must be greater than stallTickInterval.
+	if stallTickInterval := 15 * time.Second; cfg.PeerIdleTimeout <= stallTickInterval {
```

Comment:
This won't do.  What if stallTickInterval is changed in the peer package?  You should export it there instead of making a new variable.

Reply:
I agree. That is why this PR is not supposed be be reviewed without reading my comments in https://github.com/decred/dcrd/issues/383 first.

Here they are (copied from PR)

There are several possibilities for checking the validity of idle timeout against stallTickInterval, stallResponseTimeout or any other data, private to the peer package.

Do the check in config.go (as in my PR #854), hard coding the values of stallTickInterval, stallResponseTimeout.
This is not the best approach for there is a duplication of information in both config.go and peer.go.

Make stallTickInterval and stallResponseTimeout config vars as well. So config.go would have all the data it needs to do the validation but that also means that config.go somewhat 'knows' some logic which should remain private to peer.go

Make stallTickInterval and stallResponseTimeout config vars but call a validation function defined in package peer. But that would make config.go depend on peer.go.

Don't do validation in config.go but inside peer.go upon usage. I.e. in newPeerBase() but that might come a bit too late.

Are there decred's specific standards for handling these kinds of situations?

Reply:
No, there are no specific standards.  Look around at existing code.  Code what makes sense and give reasons for your changes.  You'll catch on after a few code reviews.